src/store/index.ts

```diff
@@ -76,6 +82,7 @@ const reducers = {
   [NFT_TOKENS_REDUCER_ID]: nftTokensReducer,
   [TOKEN_REDUCER_ID]: tokensReducer,
   [GATEWAY_TRANSACTIONS_ID]: gatewayTransactionsReducer,
+  [LOCAL_TRANSACTIONS_ID]: localTransactionsReducer,
```

Comment:
Did you not want to persist this state? If you refresh, it will lose the state. (If so, remember to clear the status from the store when it is moved from `queue.queue` to `history` in the `gatewayTransactions` reducer.

Reply:
In another PR.